app/proguard-rules.pro

```diff
@@ -10,4 +10,13 @@
 # Facebook Conceal proguard config
 -keep class com.facebook.crypto.** { *; }
 -keep class com.facebook.jni.** { *; }
--keepclassmembers class com.facebook.cipher.jni.** { *; }
\ No newline at end of file
+-keepclassmembers class com.facebook.cipher.jni.** { *; }
+
+# Keep generic signature of Call, Response (R8 full mode strips signatures from non-kept items).
+ -keep,allowobfuscation,allowshrinking interface retrofit2.Call
```

Comment:
should we `allowobfuscation` for retrofit? I see both keep and allowobfuscation.

Reply:
After talking with @afalls-twilio , this is the suggested rules for retrofit and it is validated to work correctly. so +1.